Makefile

```diff
@@ -25,9 +25,14 @@ GRPC_DIR:=./vendor/google.golang.org/grpc
 install: submodule
 	go install $(PACKAGES)
 
+# Generate certs for unit and release tests.
+certs:
+	go run certgen/certgen.go
+
 # Local test
-test: submodule
+test: submodule certs
 	go test -timeout 60s -race $(PACKAGES)
+	rm -f ca.crt server.crt server.key
```

Comment:
it shouldn't remove certs if they are there - we can add make clean-cert or some such if you wanted to force regen the certs but that's not necessary

Reply:
done